frontend/javascripts/oxalis/view/left-border-tabs/histogram_view.js

```diff
@@ -199,6 +278,9 @@ class Histogram extends React.PureComponent<HistogramProps, HistogramState> {
       `Enter the ${minimumOrMaximum} possible value for layer ${layerName}. Scientific (e.g. 9e+10) notation is supported.`;
 
     const minMaxInputStyle = { width: "100%" };
+    const tooltipText = isInEditMode
+      ? "Clipping in Edit Mode will also adjust the histogram's range."
+      : "Automatically clip the histogram to enhance contrast.";
```

Comment:
When being in edit mode, the tooltip doesn't explain the high-level feature. How about:

```suggestion
    const editModeAddendum = isInEditMode ?  "In Edit Mode, the histogram's range will be adjusted, too." : "";
    const tooltipText = `Automatically clip the histogram to enhance contrast.${editModeAddendum}`;
```